src/classes/frModel.cls

```diff
@@ -10,7 +10,7 @@ public with sharing abstract class frModel {
     
     public frModel(Sync_Attempt__c attempt) {
         this.syncRecord = attempt;
-        createLogRecord = attempt.Attempts__c >= frSyncRequestHandler.MAX_ATTEMPTS;
+        createLogRecord = attempt.Attempts__c >= 3;
```

Comment:
Can we revert this change so the MAX_ATTEMPTS variable is used?

Reply:
Reverted!